### DISCUSSION TITLE
Questions about  in-text citation numbers in fig.3

### DISCUSSION THREAD

**Discussion by wangkiw** [Jingkang50/OODSurvey#13] [General]
Hi~
    Thanks for your exciting work! I find some in-text citation numbers in fig.3 do not match the methods' names. For example：
1. the **MSP**  in fig.3 (a) Out-of-Distribution Detection should be  **[12] or [64]**(*A baseline for detecting misclassified and out-of-distribution examples in neural networks*) in references(two citation numbers refer to the same paper) and not [187].
2. the **IsoForest**  in (c) Anomaly Detection & Novelty Detection should be **[269]**(*Isolation forest* ) in references and not [142].

It looks like many similar mismatch errors in fig.3.

**Comment by Jingkang50** [comment]
Thanks all for pointing out the mismatch. Will fix!
